src/test/java/org/jenkinsci/plugins/gitclient/GitAPITest.java

```diff
@@ -91,6 +93,12 @@ public GitAPITest(final String gitImplName) {
         this.gitImplName = gitImplName;
     }
 
+    public static EnvVars getConfigNoSystemEnvsVars() {
```

Comment:
Would you be willing to consider using `EnvVars` rather than `EnvsVars` since it is `EnvVars` in all the other usages?

This change by itself will break compilation.  Needs to be altered for all callers as well.
```suggestion
    public static EnvVars getConfigNoSystemEnvVars() {
```